LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/NetworkServerConfiguration.cs

```diff
@@ -95,7 +129,7 @@ public static NetworkServerConfiguration CreateFromEnviromentVariables()
             config.GatewayID = envVars.GetEnvVar("IOTEDGE_DEVICEID", string.Empty);
             config.HttpsProxy = envVars.GetEnvVar("HTTPS_PROXY", string.Empty);
             config.Rx2DataRate = envVars.GetEnvVar("RX2_DATR", string.Empty);
-            config.Rx2DataFrequency = envVars.GetEnvVar("RX2_FREQ", config.Rx2DataFrequency);
+            config.Rx2Frequency = envVars.GetEnvVar("RX2_FREQ", (double?)null);
```

Comment:
As mentioned before, if we always use the double? null as default we could set null as default parameter value.
Another option is to remove the parameter and return null if not found or invalid.